src/rhos/correlatorUtils.py

```diff
@@ -2,8 +2,8 @@
 import sys
```

Comment:
Previously there were separate `correlator` and `utils` directories; this PR flattens this space.

Would it make sense to have `rhos/correlator` and `rhos/utils` directories, instead? (In that case the relative import syntax would be e.g. `from ..core` to import from further up the directory tree into this module, and `from .correlator.correlatorUtils` to import from this module into other parts of the code.)

Something to think carefully about is what can import from where. To avoid creating circular dependencies (where `A` imports `B`, `B` imports `C`, and `C` imports `A`), it's best to avoid importing from all over the place where possible. Drawing a diagram of what modules there are and where they can import from can help build up a clearer picture of the hierarchy of imports.

Reply:
I have separated the subdirectories correlator and utils. The current version works and runs. The import might be improved (and I think also reduced) in the refactoring.